src/components/HistoryVideoCard/HistoryVideoCard.js

```diff
@@ -0,0 +1,49 @@
+import { faTrash } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React from 'react'
+import { useNavigate } from 'react-router-dom'
+import { useState } from 'react'
+import { useData } from '../../Context/Context'
+import { useAuth } from '../../Context/AuthProvider'
+import { removeFromHistory } from '../../Utility/historyService'
+import './HistoryVideoCard.css'
+
+const HistoryVideoCard = ({ videoItem }) => {
+    const navigate = useNavigate()
+    const { watchlater, likedVideos, watchedVideos } = useData()
+    const { token } = useAuth()
+    const { showPlaylistModal, setCurrentVideo, dispatch } = useData()
+    const [showCTAcontainer1, setShowCTAcontainer] = useState(false)
+
+    const openVideoPage = (e) => {
+        e.preventDefault();
+        if (e.target.classList.value !== 'svg-inline--fa fa-ellipsis-vertical cta-icon') {
+            navigate(`/video/${videoItem._id}`)
+        }
+    }
+
+    return (
+        <>
+            <div className='video-item-container flex-vt' >
```

Comment:
A fragment is used when you have multiple sibling elements and you need to wrap them around a parent but don't need an extra element.

Here, your div with video-item-container is already a parent here. I don't see the purpose of using a fragment here.
Any reason?